src/Controllers/CheckoutController.php

```diff
@@ -261,31 +304,44 @@ public function doAuth(
 
     /**
      * @param PaymentService $paymentService
-     * @param $ordeId
-     *
+     * @param int $orderId
      * @return string
+     * @throws \Throwable
      */
     public function doAuthFromOrder(
         PaymentService $paymentService,
-        $orderId
+        int $orderId
     ) {
 
-        /** @var OrderRepositoryContract $orderContract */
-        $orderContract = pluginApp(OrderRepositoryContract::class);
+        /** @var OrderHelper $orderHelper */
+        $orderHelper = pluginApp(OrderHelper::class);
 
-        /** @var \Plenty\Modules\Authorization\Services\AuthHelper $authHelper */
-        $authHelper = pluginApp(AuthHelper::class);
+        $order = $orderHelper->getOrderByOrderId($orderId);
```

Comment:
getOrderByOrderId is deprecated

```suggestion
        $order = $orderHelper->findById($orderId);
```